server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodesStatsRequest.java

```diff
@@ -219,7 +219,8 @@ public enum Metric {
         RESOURCE_USAGE_STATS("resource_usage_stats"),
         SEGMENT_REPLICATION_BACKPRESSURE("segment_replication_backpressure"),
         REPOSITORIES("repositories"),
-        ADMISSION_CONTROL("admission_control");
+        ADMISSION_CONTROL("admission_control"),
+        SEARCH_CACHE_STATS("caches");
```

Comment:
Ah, I thought you meant just the variable name. I think we were thinking "caches" in the URI because we might expand this API to cover other caches than just the request_cache in future, but not sure if it would still be limited to search caches.

Reply:
I am not in favor of renaming this to ```search_cache``` as these caches(tiered/disk) can be easily extended to indexing and other areas. I can't think of a better generic term other than ```caches``` but open to other suggestions.